libs/books/data-access/src/lib/+state/reading-list.effects.ts

```diff
@@ -2,9 +2,11 @@ import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
 import { Actions, createEffect, ofType, OnInitEffects } from '@ngrx/effects';
 import { of } from 'rxjs';
-import { catchError, concatMap, exhaustMap, map } from 'rxjs/operators';
-import { ReadingListItem } from '@tmo/shared/models';
+import { catchError, concatMap, exhaustMap, map, tap, switchMap } from 'rxjs/operators';
```

Comment:
Newly added operator are unused in this file.
Please remove `tap `and `switchMap` from here.

Reply:
Removed unused operators